rotkehlchen/db/dbhandler.py

```diff
@@ -1246,15 +1246,19 @@ def get_tokens_for_address(
             cursor: 'DBCursor',
             address: ChecksumEvmAddress,
             blockchain: SupportedBlockchain,
+            token_exceptions: set[ChecksumEvmAddress],
     ) -> tuple[list[EvmToken] | None, Timestamp | None]:
         """Gets the detected tokens for the given address if the given current time
         is recent enough.
 
         If not, or if there is no saved entry, return None.
         """
-        ignored_asset_ids = self.get_ignored_asset_ids(cursor)
         last_queried_ts = None
-        querystr = 'SELECT key, value FROM evm_accounts_details WHERE account=? AND chain_id=? AND (key=? OR key=?)'  # noqa: E501
+        querystr = (
+            'SELECT key, value FROM evm_accounts_details WHERE account=? AND chain_id=? '
+            'AND (key=? OR key=?) AND value NOT IN '
+            '(SELECT value FROM multisettings WHERE name="ignored_asset")'
```

Comment:
why add this here? Is it specific to this PR? It's not right? You seemto move the query from `ignored_asset_ids` here in the DB query. It's probably better/more performant right? That's why you are doing this?

But again not something specific for monerium?

Reply:
is not related to the PR. I made the change because I added a variable token_exceptions and then I saw that we were querying the ignored assets for each address in a loop and then I said... Okey then the ignored assets should be also in the argument and I should just combine the ignored assets and the exceptions... but then I looked at it and said... fuck we can do it better in sqlite and don't load to many things into memory. So is not related but I believe it's much better both for readibility and performance